person.rb

```diff
@@ -1,6 +1,9 @@
 require_relative 'nameable'
 
 class Person < Nameable
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
   def initialize(age, name = 'Unknown', parent_permission = 'true')
```

Comment:
- [ ] I am unable to create a student kindly fix this issue.
![image](https://user-images.githubusercontent.com/81932478/204108777-fdc3d2f3-e9b5-4255-b919-39b6b6d60340.png)